docs/ru/clover-swarm.md

```diff
@@ -0,0 +1,31 @@
+# Название проекта
+
+[CopterHack-2022](copterhack2022.md), команда **%Team_name%**.
```

Comment:
Is this correct name of your team?

Reply:
> Is this correct name of your team?

Actually no. We will change the team name as soon as we come up with it.